tests/test_invoice.py

```diff
@@ -98,6 +98,17 @@ async def test_send_required_args_only(self, bot, chat_id, provider_token):
         assert message.invoice.title == self.title
         assert message.invoice.total_amount == self.total_amount
 
+        link = await bot.create_invoice_link(
+            title=self.title,
+            description=self.description,
+            payload=self.payload,
+            provider_token=provider_token,
+            currency=self.currency,
+            prices=self.prices,
+        )
+        assert isinstance(link, str)
+        assert link != ""
```

Comment:
what does the actual invoice link look like?

Reply:
https://t.me/invoice/UniqueLettersAndNumbers